scripts/process-g6-into-elastic-search.py

```diff
@@ -60,6 +60,173 @@ def category_name_to_id(name):
+
+    # #########################
+    # ## service management ###
+    # #########################
+
+    # support types is array in G6, boolean in G5
+    # - any G6 value sets G5 to true
+    if "supportTypes" in data and len(data["supportTypes"]) > 0:
+        attributes.append({"name": "q25",  "q25": "true"})
+
+    attributes.append(boolean_attribute("serviceOnboarding", "q26",
+                                        attributes, data))
+    attributes.append(boolean_attribute("serviceOffboarding", "q27",
+                                        attributes, data))
+    attributes.append(boolean_attribute("dataExtractionRemoval", "q28",
+                                        attributes, data))
+    attributes.append(boolean_attribute("datacentresEUCode", "q31",
+                                        attributes, data))
+    attributes.append(boolean_attribute("dataBackupRecovery", "q36",
+                                        attributes, data))
+    attributes.append(boolean_attribute("selfServiceProvisioning", "q39",
+                                        attributes, data))
+    attributes.append(boolean_attribute("supportForThirdParties", "q41",
+                                        attributes, data))
+    attributes.append(boolean_attribute("supportForThirdParties", "q41",
+                                        attributes, data))
+
+    if "datacentreTier" in data:
+        attributes.append({"name": "q32",  "q32": data["datacentreTier"]})
+
+    # ## Data centre tiers:
+    # ## map G6 to these
+
+    if "datacentreTier" in data:
+        if "TIA-942 Tier 1" in data["datacentreTier"]:
+            attributes.append({"name": "q32",  "q32": "tier1tia942"})
+        if "Uptime Institute Tier 1" in data["datacentreTier"]:
+            attributes.append({"name": "q32",  "q32": "tier1uptimeinstitute"})
+        if "TIA-942 Tier 2" in data["datacentreTier"]:
+            attributes.append({"name": "q32",  "q32": "tier2tia942"})
+        if "Uptime Institute Tier 2" in data["datacentreTier"]:
+            attributes.append({"name": "q32",  "q32": "tier2uptimeinstitute"})
+        if "TIA-942 Tier 3" in data["datacentreTier"]:
+            attributes.append({"name": "q32",  "q32": "tier3tia942"})
+        if "Uptime Institute Tier 3" in data["datacentreTier"]:
+            attributes.append({"name": "q32",  "q32": "tier3uptimeinstitute"})
+        if "TIA-942 Tier 4" in data["datacentreTier"]:
+            attributes.append({"name": "q32",  "q32": "tier4tia942"})
+        if "Uptime Institute Tier 4" in data["datacentreTier"]:
+            attributes.append({"name": "q32",  "q32": "tier4uptimeinstitute"})
+
+    # provising time is 1 Day etc in G6, but the question is
+    # "documented?" in G5 so any value provokes true
+    if "provisioningTime" in data:
+        attributes.append({"name": "q40",  "q40": "true"})
+
+    # ##################
+    # ## PaaS / IaaS ###
```

Comment:
Isn't this just PaaS (lot 2), with the IaaS versions of the same questions below?  Should it be conditional on the lot of the current G6 service which of the two filters gets added, rather than adding filters for two different lots to a single service?